client/header/activity-panel/index.js

```diff
@@ -260,14 +260,36 @@ class ActivityPanel extends Component {
 }
 
 export default withSelect( select => {
-	const { getReportItems, getReportItemsError, isReportItemsRequesting } = select( 'wc-api' );
+	const {
+		getCurrentUserData,
+		getNotes,
+		getNotesError,
+		getReportItems,
+		getReportItemsError,
+		isGetNotesRequesting,
+		isReportItemsRequesting,
+	} = select( 'wc-api' );
 	const orderStatuses = wcSettings.wcAdminSettings.woocommerce_actionable_order_statuses || [
 		'processing',
 		'on-hold',
 	];
+	const userData = getCurrentUserData();
+
+	const notesQuery = {
+		page: 1,
+		per_page: 1,
+	};
+
+	const latestNote = getNotes( notesQuery );
+	const unreadNotes =
```

Comment:
@joshuatf do you think this logic might also have a home in PHP - thinking down the road if/when mobile apps want to consume the inbox notifs - they too will need the ability to set an unread indicator.